### PR TITLE
Updates to vocabulary.  Alphabetized entries.

### PR DESCRIPTION
**Pull request by stumbo**

### CODE REVIEW

File: content/en/medley/project/vocabulary/_index.md [hunk 1]
```diff
@@ -8,17 +8,44 @@ The names “Medley” “Medley Interlisp” “Interlisp” “Interlisp.org
...
-Interlisp.org
-: a “domain name”, used for some web sites and email addresses
+Alto Lisp
+:  An (unsuccessful) attempt to build a Lisp-based OS for running Interlisp on an Alto
 
-- <https://interlisp.org>  – the address of the main web site
-- <https://online.interlisp.org> – the address of a service that lets you run Medley
-- <info@interlisp.org>             - an email address of a group that can respond to questions
+Carol, Fugue, Harmony, Intermezzo, Koto, Lyric, Medley
+: Named releases of Interlisp-D.  All are obsolete except Medley.
```
Comment (masinter) [inline content/en/medley/project/vocabulary/_index.md:15]: I'm not sure what it means to say they are "obolete". How is this helpful?

Comment (stumbo) [reply content/en/medley/project/vocabulary/_index.md:15]: I was in the process of breaking this list into individual entries, you'll note later on there are entries like:

```
Fugue
: An obsolete named release of Interlisp-D.
```

I neglected to go back and clean up the first entry.  But, the wording is the original wording used in the vocabulary.  My intent was to lightly edit the wording.  I'm open to other suggestions on what the appropriate wording for any entry in the vocabulary should be.

**Comment by masinter** [comment]
I wouldn't pay that much attention to the "origianl" vocabulary -- it was something I jotted down, we should fix it.
- [ ] "Glossary" might be better than "Vocabulary"
- [ ] Fugue, Harmony, Lyric, Koto etc.: give date (year, month), Naybe in the timeline

**Comment by masinter** [comment]
and be sure we're using the terminology we established. "Building the Medley system" rather than "Building the Interlisp system".
I'd suggest searching for the terms we're realigning and make sure usage is consistent: Interlisp, Interlisp-D, MEdley, the Medley Interlisp Project (unlike before we're only using "Medley Interlisp" for the project)

**Pull request merged**